packages/server/lib/screenshots.ts

```diff
@@ -468,8 +487,10 @@ module.exports = {
 
       return getBuffer(details)
       .then((buffer) => {
+        // @ts-ignore
         return fs.outputFileAsync(pathToScreenshot, buffer)
       }).then(() => {
+        // @ts-ignore
         return fs.statAsync(pathToScreenshot).get('size')
```

Comment:
I think this will work if we want to keep the same Bluebird Promise pattern. Otherwise we could 

```suggestion
        return (fs.statAsync(pathToScreenshot) as Bluebird.Promise<Stats>).get('size')
```

or we could convert the then to:

```
.then(async () => {
  return (await fs.stat(pathToScreenshot)).size
})
```

Reply:
or use the native async fs module - `import { stat } from 'fs/promises` and `return (await stat(path)).size`

Reply:
[`7d6e3a0` (#30758)](https://github.com/cypress-io/cypress/pull/30758/commits/7d6e3a0b960a487f596937ecb85371a3e39534eb)